### PR TITLE
Jalalhabeeb browser week1

### PR DESCRIPTION
**Pull request by JalalHabeeb**

### CODE REVIEW

**Review by kasimoglou — CHANGES_REQUESTED** [review]
Well done Jalal! The first 4 exercises work perfectly, the 5th needs some more work but you are in the right direction!

Test results are not included in the PR, I run them locally and I saw that they fail for the second exercise as the `body` element does not have the required font family. Could you please include this fix?

Also, this MR contains exercises from previous weeks. It would be very helpful if the next time you could create a brand new branch from the main branch and include there only the changes that are relevant to the corresponding week. Please let me know if you need help on how exactly you could do that.

File: 2-Browsers/Week1/homework/ex2-aboutMe/index.js [hunk 1]
```diff
@@ -8,5 +8,14 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
    `list-item`.
 3. Look in the css file!
 ------------------------------------------------------------------------------*/
+const nickName = document.querySelector('#nickname');
+nickName.textContent = 'Jay';
+const favFood = document.querySelector('#fav-food');
+favFood.textContent = 'Pizza';
+const homeTown = document.querySelector('#hometown');
+homeTown.textContent = 'Homs';
```
[the remark this answers is not in the document]
Comment (JalalHabeeb) [reply 2-Browsers/Week1/homework/ex2-aboutMe/index.js:16]: Yes indeed, I should've followed DRY concept.

File: 2-Browsers/Week1/homework/ex4-whatsTheTime/index.js [hunk 2]
```diff
@@ -8,7 +8,16 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
...
-  // TODO complete this function
+  const container = document.createElement('div');
+  document.body.appendChild(container);
+  container.style.textAlign = 'center';
+  container.style.margin = '0px auto';
+  container.style.marginTop = '200px';
+  setInterval(() => {
+    console.clear();
+    const time = new Date();
+    container.textContent = `Local Time is: ${time.getHours()}:${time.getMinutes()}:${time.getSeconds()}`;
```
[the remark this answers is not in the document]
Comment (JalalHabeeb) [reply 2-Browsers/Week1/homework/ex4-whatsTheTime/index.js:19]: I will of course, but anyway this Date object still confuses me everytime I deal with it.

File: 2-Browsers/Week1/homework/ex5-catWalk/index.js [hunk 3]
```diff
@@ -21,8 +21,16 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
...
 -----------------------------------------------------------------------------*/
+
 function catWalk() {
-  // TODO complete this function
+  const image = document.querySelector('img');
+  image.style.left = '0px';
+  let currentLeft = parseInt(image.style.left);
+  setInterval(() => {
+    if (currentLeft === image.style.left) {
+      currentLeft = image.style.left + 10 + 'px';
```
[the remark this answers is not in the document]
Comment (JalalHabeeb) [reply 2-Browsers/Week1/homework/ex5-catWalk/index.js:31]: This whole solution is a mess I know. I was working on it and it was moving but not repeating when reach the edge, so I deleted everything and started over.
I'll work on it.

**Comment by JalalHabeeb** [comment]
About the body font family, this is a test that remained from another test method, it's an error in the test itself, I told Rob about it and I think he fixed it.
About the exercises from another week, they are here because in week 1 I didn't make a branch and worked directly in the main branch, and now I don't know how to fix this. can you help me with it?

**Review by kasimoglou — APPROVED** [review]

**Pull request closed without merging**